### PR TITLE
Fix bug with calling waterline create with a buffer

### PR DESCRIPTION
**Pull request by benbp**
Fixes https://github.com/RackHD/RackHD/issues/40

@heckj @kacole2

### CODE REVIEW

**Comment by benbp** [comment]
I deleted the hound errors (line too long, 'Buffer' not defined) because they are false negatives.

**Comment by rolandpoulter** [comment]
:+1:

**Comment by tldavies** [comment]
+1

File: lib/common/template.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
             Constants.Templates.Directory
         ).then(function (templates) {
             var promises = _.map(templates, function (contents, name) {
-                return self.put(name, contents);
+                return self.put(name, contents.toString());
```
Comment (rolandpoulter) [inline lib/common/template.js]: Only concern here is that this will break if context is null or undefined. I'm assuming that is not possible here.

Comment (benbp) [inline lib/common/template.js]: Yeah I think worst case we'll get an empty buffer out of the fs library.

Comment (benbp) [inline lib/common/template.js]: Yeah, the fs library will give us an empty buffer for an empty file. If the file does not exist, we'll have failed before we get to this point:

```
> e = fs.readFileSync('./nothere')
Error: ENOENT, no such file or directory './nothere'
    at Object.fs.openSync (fs.js:432:18)
    at Object.fs.readFileSync (fs.js:289:15)
    at repl:1:9
    at REPLServer.self.eval (repl.js:110:21)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.emit (events.js:95:17)
    at Interface._onLine (readline.js:202:10)
    at Interface._line (readline.js:531:8)
    at Interface._ttyWrite (readline.js:760:14)
    at ReadStream.onkeypress (readline.js:99:10)
> e = fs.readFileSync('./emptyfile')
<Buffer >
> e.toString()
''
```

**Comment by kacole2** [comment]
thanks. i can test it here soon

**Pull request merged**